sdk/java-sdk/src/main/scala/kalix/javasdk/impl/EntityExceptions.scala

```diff
@@ -76,6 +77,9 @@ object EntityExceptions {
 
     def apply(init: ReplicatedEntityInit, message: String): EntityException =
       ProtocolException(init.entityId, message)
+
+    def apply(init: WorkflowInit, message: String): EntityException =
```

Comment:
we may want to change the protocol once again to align here. `WorkflowInit` should become `WorkflowEntityInit`